src/Lykke.Job.TransactionHandler/Queues/CashInOutQueue.cs

```diff
@@ -13,102 +14,108 @@
 
 namespace Lykke.Job.TransactionHandler.Queues
 {
-    public sealed class CashInOutQueue : IQueueSubscriber
+    public sealed class CashInOutQueue : IStartable, IDisposable
     {
         private const bool QueueDurable = true;
 
         private readonly ILog _log;
+        private readonly string _mongoConnectionString;
```

Comment:
I would prefer to use RabbitMqSettings and MongoDeduplicator instead of lot of strings.